torch/csrc/cuda/comm.cpp

```diff
@@ -129,7 +129,7 @@ std::vector<at::Tensor> scatter(
   std::vector<at::Tensor> chunks;
   if (chunk_sizes) {
     const int64_t chunk_size_sum =
-        std::accumulate(chunk_sizes->begin(), chunk_sizes->end(), 0);
+        std::accumulate(chunk_sizes->begin(), chunk_sizes->end(), int64_t{0});
```

Comment:
Honest question: do we prefer this way of spelling it over 0ll?

Reply:
More explicit always happy?